index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone Module1</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="about.html">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="icon" href="./Images/logo-no-background.svg" type="image/svg" />
+  </head>
+  <body>
+    <section class="nav-bar">
+      <div id="ham-menu">
+        <img
+          onclick="clickX()"
+          id="x"
+          src="https://img.icons8.com/sf-regular/48/FFFFFF/multiply.png"
+        />
+        <ul class="ham">
+          <li>
+            <a href="#" class="ham-items" onclick="clickX()"
+              >Home</a
+            >
+          </li>
+          <li>
+            <a href="about.html" class="ham-items" onclick="clickX()">About</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Program</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Join</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Sponsor</a>
+          </li>
+        </ul>
+      </div>
+      <header class="head flex-row">
+        <nav>
+          <img
+            onclick="clickMenu()"
+            id="menu"
+            src="./Images/Icon - Menu.png"
+            alt="Menu Logo"
+            class="menu-logo"
+          />
+          <a href="">
+            <img class="logo" src="Images/logo-black.png" alt="logo" />
+          </a>
+          <ul class="menu_items flex-row">
+            <li class="nav_items"><a href="#">Home</a></li>
+            <li class="nav_items"><a href="about.html">About</a></li>
```

Comment:
- [ ] you are missing the The top short bar section with social media links, logout, and language links aligned to the right.please implement for both pages.

yours
![Screenshot (976)](https://user-images.githubusercontent.com/65298048/233434050-117dfd00-bd36-401e-bd00-67f58d0f2cc5.png)
original

![Screenshot (975)_LI](https://user-images.githubusercontent.com/65298048/233434238-a8d25adf-b603-4102-a865-8c1927cb9426.jpg)